lesson002/task2/styles/2style.css

```diff
@@ -0,0 +1,39 @@
+body{
+    margin: 0;
+    padding: 0;
+}
+.div1 {
+    width:100%;
+    min-height:600px;
+}
+.head {
+    width:100%; 
+    height:150px; 
+    background-color: rgba(0, 255, 255, 0.607);
+}
+.block {
+    width:100%; 
+    height:90px; 
+    background-color: rgba(252, 155, 95, 0.995);
+}
+.div2 {
+    width:100%;
+    min-height:700px;
+}
+.menu {
+    width:30%; 
+    height:700px; 
+    background-color: rgb(118, 197, 102); 
+    float:left;
+}
+.content {
+    width:70%; 
+    height:700px; 
+    background-color: rgb(151, 171, 171); 
+    float:left;
+}
+.podval {
+    width:100%; 
+    height:100px; 
+    background-color: rgb(29, 18, 253);
```

Comment:
Use the height ratio too with the help of percentages. Footer not visible on my screen

Reply:
Done